.drone.yml

```diff
@@ -388,6 +388,19 @@ steps:
     commands:
       - git fetch --tags --force
 
+  - name: build
+    pull: always
+    image: golang:1.13
+    commands:
+      - curl -sL https://deb.nodesource.com/setup_12.x | bash - && apt -y install nodejs
+      - make css
+      - make js
+      - make generate
```

Comment:
Didn't we get rid of make generate?

Reply:
`make build` runs make generate before building, but in the case of xgo it was included in the `release` step I just moved it back out again.

Thinking about this, I could probably put it back in release, and change the image that css/js use to speed up build.

Reply:
Please see update commits for changes.